packages/erlang/build.sh

```diff
@@ -20,8 +20,8 @@ termux_step_post_get_source() {
 
 termux_step_host_build() {
 	cd $TERMUX_PKG_SRCDIR
-	./configure --enable-bootstrap-only
-	make -j "$TERMUX_MAKE_PROCESSES"
+	./configure --enable-bootstrap-only --host=$TERMUX_HOST_PLATFORM --build=$TERMUX_BUILD_TUPLE
```

Comment:
Why you are doing this for **host** build? These flags should be passed only for cross-compilation steb, but in termux_step_host_build you need to compile erlang for Ubuntu x86_64.

Reply:
Probably we don't need to host build Erlang. https://git.alpinelinux.org/aports/tree/community/erlang/APKBUILD

See Alpine Linux APKBUILD file. Passing appropriate `--host` and `--build` should do